src/current/v23.2/advanced-changefeed-configuration.md

```diff
@@ -53,9 +82,9 @@ We do **not** recommend adjusting these settings unless you are running a large
 
 **Default:** `3s`
 
-This setting controls the frequency of checkpoints for each [range]({% link {{ page.version.version }}/architecture/overview.md %}#range). A changefeed aggregates these checkpoints across all ranges, and once the timestamp on all the ranges advances, the changefeed can then [checkpoint]({% link {{ page.version.version }}/how-does-an-enterprise-changefeed-work.md %}). As a result, the higher the value of this setting the longer it can take for a changefeed to checkpoint. It is important to note that a changefeed at default configuration does not checkpoint more often than once every 30 seconds.
+This setting controls the frequency of checkpoints for each [range]({% link {{ page.version.version }}/architecture/overview.md %}#range). A changefeed aggregates these checkpoints across all ranges, and once the timestamp on all the ranges advances, the changefeed can then [checkpoint]({% link {{ page.version.version }}/how-does-an-enterprise-changefeed-work.md %}). As a result, the higher the value of this setting the longer it can take for a changefeed to checkpoint. It is important to note that a changefeed at default configuration does not checkpoint more often than once every 30 seconds (configurable with the [`min_checkpoint_frequency`]({% link {{ page.version.version }}/create-changefeed.md %}#min-checkpoint-frequency) option).
 
-In clusters running large-scale workloads, increasing this setting can help to lower the potential impact of changefeeds on SQL latency. That is, an increase in the setting could lower the load on the cluster. This is important for workloads with tables in the TB range of data. However, for most workloads, we recommend leaving this setting at the default of `3s`.
+In clusters running large-scale workloads, increasing `kv.closed_timestamp.target_duration` can help to lower the potential impact of changefeeds on SQL latency. That is, an increase in the setting could lower the load on the cluster. This is important for workloads with tables in the TB range of data. However, for most workloads, we recommend leaving this setting at the default of `3s`.
```

Comment:
This explanation for `closed_timestamp.target_duration` is this still valid? Let me know if this should be updated.

Reply:
Sigh.. I think we need to mention  kv.rangefeed.closed_timestamp_refresh_interval 

Basically the above is the "big hammer" -- it works; but if you also use follower
reads, you may not be able to set kv.closed_timestamp.target_duration to high value.  In that case, bump up kv.rangefeed.closed_timestamp_refresh_interval  instead.  We really need to get some kv folks to re-review this though.